template/.ruby-version

```diff
@@ -1 +0,0 @@
-2.7.5
```

Comment:
We still need this file to stay in sync inside the team which Ruby version we're all using.